src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -450,11 +465,11 @@ class GroupReadsByUmi
     logger.info("Filtering and sorting input.")
     in.iterator
       .filter(r => !r.secondary && !r.supplementary)
-      .filter(r => (includeNonPfReads || r.pf)            || { filteredNonPf += 1; false })
-      .filter(r => (r.paired && r.mapped && r.mateMapped) || { filteredPoorAlignment += 1; false })
-      .filter(r => (r.refIndex == r.mateRefIndex)         || { filteredPoorAlignment += 1; false })
-      .filter(r => (r.mapq >= this.minMapQ && r.get[Int](SAMTag.MQ.name()).exists(_ >= this.minMapQ)) || { filteredPoorAlignment += 1; false })
-      .filter(r => !r.get[String](rawTag).exists(_.contains('N')) || { filteredNsInUmi += 1; false })
+      .filter(r => (includeNonPfReads || r.pf)                      || { filteredNonPf += 1; false })
+      .filter(r => (r.mapped && (r.unpaired || r.mateMapped))       || { filteredPoorAlignment += 1; false })
+      .filter(r => (r.unpaired || r.refIndex == r.mateRefIndex)     || { filteredPoorAlignment += 1; false })
```

Comment:
Did you want to remove this filter?

Reply:
I was going to leave it for now as you were a little concerned about it.  We can always remove it in a separate PR.